### PR TITLE
PE 546 SFI Page Redesign

### PR DESCRIPTION
**Pull request by vuphuctho**
- [x] Redo SFI design
- [x]  UI responsiveness (mobile display for non-responsive components like Tab)
- [x] Display correct data from backend

### CODE REVIEW

**Comment by shuesken** [comment]
Great @vuphuctho! Noticing some superficial issues looking at [`caf`:](http://localhost:3000/sustainable-eco-banks/caf)

---

Serving shouldn't display `[]` when empty (although I don't know what it _should_ display)
image

----

It should say "Mortgage **or** Loans"
image

---

Spacing between bullet points is too tight compared to the mockup:
image

--- 

The stock image aren't added yet either, but I believe that's addressed that in the Linear comment from albert: https://linear.app/bankgreen/issue/PE-546#comment-a992f2d4

---

I don't see a mockup for the mobile page, but to me, this spacing between the sections looks too big:
image

---

I think it'd be useful to have consistency between kebab-case and camelCase in the component prop passing both in `EcBankHeader` and `EcoBankDetail`

**Comment by vuphuctho** [comment]
@shuesken 

> - Serving shouldn't display [] when empty (although I don't know what it should display)
> - It should say "Mortgages or Loans"
> - Spacing between bullet points is too tight compared to the mockup:

I have fixed above issues in recent commit

> I think it'd be useful to have consistency between kebab-case and camelCase in the component prop passing both in `EcoBankHeader` and `EcoBankDetail`

Since we use camelCase in our code, I have modified all kebab-case to camelCase

**Comment by vuphuctho** [comment]
> Spacing between bullet points is too tight compared to the mockup:
Added the styling fix. Though if we can control it via Prismic, I think it would be better.

**Comment by vuphuctho** [comment]
@shuesken @RogerTangos For page without prismic content, for now we display `fromTheWebsite` like so:

page URL: http://localhost:3000/sustainable-eco-banks/amalgamated_bank
Current design
Screenshot 2023-05-12 at 14-47-07 Find Ethical   Sustainable Banks In Your Area - Bank Green

New design 
Screenshot 2023-05-12 at 14-46-44 Amalgamated Bank Review and Service Offering - Bank Green

**Comment by shuesken** [comment]
@vuphuctho Lovely! I think the "Serving: []" thing is still there though. Please also remove the "Tell us if we got something wrong", it seems odd to have there. (This is just my design sense talking tho)

Otherwise looks good to me! Would love to @RogerTangos 's check on the design too, but I think we're good to go here otherwise.

Out of curiosity, why would you prefer having the styling in Prismic rather than in the frontend? Addding the `space-y-6` the way you did seems to me like exactly the right way of separating content and website design there.

**Comment by vuphuctho** [comment]
@shuesken Sorry, my misunderstaning about 'Serving'. I have fixed it in last commit

About styling of Prismic content, right now we limit these fields to be bullet points, so adding `space-y-6` solves our styling requirement. But in the future, if we need to enable more content types in these section (paragraphs, multi-leveled bullet points) then refining the styling would be a hassle. If the content creator on Prismic does not need to contact our dev side if they want to introduce any new type of content but can fully control from their side, it would be better for them.

However, just my two cent. Since our team is small, I don't see it as an important requirement.

**Comment by RogerTangos** [comment]
I'm still seeing the issue with "serving" here. Zak and I talked about this after refining the ticket and I made the mistake of not re-editing it. **Instead of using "serving" from django, can we please use whatever text is currently in prismic?** That's probably easier to implement than using the data from django anyways, it separates concerns, and gives content editors more flexibility 

Screenshot 2023-05-04 at 10 19 54

**Comment by RogerTangos** [comment]
Another issue: I believe that the accredidation images (with the exception of fossil free certified) should now be served via the prismic SFIDefaults types. i.e. `institution_credentials-bankers for net zero`

**Comment by RogerTangos** [comment]
I fixed the "serving" issue. The accredidations still look a bit wonky to me though. For example, Unity Trust Bank is both a part of GABV and Bankers for Net Zero, but the image doesn't appear for me. I'd prefer to get the `ecoBankHeader` displaying accredidations directly from the SFIDefaults page if possible, but they really should work at launch.

File: components/eco-bank/EcoBankHeader.vue [hunk 1]
```diff
@@ -23,22 +29,27 @@
                     <span v-else-if="rating === 'ok'">This bank is ok</span>
                     <span v-else-if="rating === 'great'">This bank is great</span>
                 </div>
+                <div class="flex justify-start items-center gap-6 mt-4" v-if="hasInstitutionCredentials">
```
Comment (RogerTangos) [inline components/eco-bank/EcoBankHeader.vue:32]: should use prismic if possible. Alt should also be pulled from prismic

**Review by RogerTangos — CHANGES_REQUESTED** [review]

**Comment by vuphuctho** [comment]
@RogerTangos I reverted [your last change]([https://github.com/bank-green/nuxt-bank-green/pull/48/commits/e17779114a3978f1da6dde3b4e0383f6cc43be2a) in order to display `serving` with `PrismicText` instead.

**Comment by vuphuctho** [comment]
@RogerTangos the accreditation images are now used Prismic resource instead of local ones.

**Review by RogerTangos — APPROVED** [review]

**Comment by RogerTangos** [comment]
Approved. DO NOT MERGE YET. Merging will not take place until other in-sprint tickets have been completed.

**Pull request merged**